LoginAndAclSetup.md

```diff
@@ -0,0 +1,160 @@
+# Setting up optional login and document level access control
+
+## Table of Contents
+
+- [Requirements](#requirements)
+- [Setting up Azure AD Apps](#setting-up-azure-ad-apps)
+  - [Manual Setup](#manual-setup)
```

Comment:
Unfortunately missing from this PR - automatic setup. It's quite the can of worms because it involves calling the graph API, but we are now pulling in the graph sdk which should make this simpler. Can automatic setup wait until a follow up PR? @pamelafox  has a similar script in her PR #534

Reply:
follow up pr